mod-password-validator/src/test/resources/global/add-okapi-permissions.feature

```diff
@@ -0,0 +1,81 @@
+Feature: prepare data for api test
```

Comment:
This file need only for local testing, you can remove it. Generally, we run tests like that:
```
-Dtest=QuickMarcApiTest -DargLine="-Dkarate.env=testing"
```